include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -696,6 +696,18 @@ typedef void (*OrtCustomJoinThreadFn)(OrtCustomThreadHandle ort_custom_thread_ha
 
 typedef OrtStatus*(ORT_API_CALL* RegisterCustomOpsFn)(OrtSessionOptions* options, const OrtApiBase* api);
 
+/** \brief Callback function for RunAsync
+ *
+ * \param[in] user_data A customized handle passed in by RunAsync.
```

Comment:
The word 'handle' might be confusing. Can we just say "user specific data that will be passed back to the callback"?

Reply:
Done.

Reply:
let's drop it.